log/log.go

```diff
@@ -0,0 +1,38 @@
+package log
+
+import (
+	"github.com/sirupsen/logrus"
+	"log"
+)
+
+type UseCaseLogger interface {
+	Log(args ...any)
```

Comment:
Figured I'd leave this interface as bland as possible. The wrappers themselves can add decoration etc